applications/orbovideosr/README.md

```diff
@@ -0,0 +1,90 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 ORBO
+```
+
+
+# **Orbo VideoSR Application Overview**
+Orbo Video Super Resolution converts a low resolution video into high resolution video.

```

Comment:
Can you pack a little more information in overview. For example taking some literature from your website like "Cutting edge computer vision and deep learning technology is used to build 2X Super resolution algorithm which provides high definition images with improved details"